src/database/sql_database.cc

```diff
@@ -966,7 +966,7 @@ std::vector<std::shared_ptr<CdsObject>> SQLDatabase::browse(BrowseParam& param)
             for (std::size_t count = 0; count < dynContent->size(); count++) {
                 auto dynConfig = dynContent->get(count);
                 if (parent->getLocation() == dynConfig->getLocation().parent_path() || (parent->getLocation().empty() && dynConfig->getLocation().parent_path() == "/")) {
-                    int dynId = -(parent->getID() + 1) * 10000 - count;
```

Comment:
If you cast count to `int` the warning should be gone as well and the code is clearer.

Anyway, you accidently added this commit twice.

Reply:
Still complains. Will remove the second commit.